client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -0,0 +1,216 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { isExternal } from 'lib/url';
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+import Sidebar from 'layout/sidebar';
+import SidebarFooter from 'layout/sidebar/footer';
+import SidebarItem from 'layout/sidebar/item';
+import SidebarMenu from 'layout/sidebar/menu';
+import SidebarRegion from 'layout/sidebar/region';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const SECTIONS = [
```

Comment:
I think you were right with the previous approach with the menu sections and items. I'm reconsidering this approach now. 😅

Reply:
No worries 😀It's reverted in adebbf5.